website/content/docs/secrets/transit.mdx

```diff
@@ -232,6 +232,57 @@ the proper permission, it can use this secrets engine.
     data, since the process would not be able to get access to the plaintext
     data.
 
+## BYOK
```

Comment:
Lets be explicit here: "Bring Your Own Key (BYOK)"

Reply:
Also, I'd like some language discouraging BYOK when not needed.  Something like "While creating a key within the Transit Secrets Engine and letting Vault create and manage the key material is more secure, there may be a need to bring a key from an existing HSM or system.  The key import functionality enables this case."